WalletWasabi/WabiSabi/Client/CredentialDependencies/DependencyGraph.cs

```diff
@@ -176,16 +176,15 @@ private DependencyGraph AddOutput(IEnumerable<ulong> values)
 		/// input side the structure fans in so necessarily it leaves no nodes
 		/// with non-maximized in-degrees, it can only increase the available
 		/// out degree for nodes which are not fully consumed.</para></remarks>
-		private DependencyGraph ResolveCredentials()
-			=> CredentialTypes.Aggregate(
-				CredentialTypes.Aggregate(this, (g, credentialType) => g.ResolveNegativeBalanceNodes(credentialType)),
-				(g, credentialType) => g.ResolveZeroCredentials(credentialType));
+		private DependencyGraph ResolveCredentials() =>
+			ResolveNegativeBalanceNodes(CredentialType.Amount).ResolveNegativeBalanceNodes(CredentialType.Vsize)
+			.ResolveZeroCredentials(CredentialType.Amount).ResolveZeroCredentials(CredentialType.Vsize);
```

Comment:
The problem with this change is that it is not exactly the same because if tomorrow we need to add a new kind of credential the existing code will work while the new code will not.

In case this is okay then could format it as follow?

```suggestion
		private DependencyGraph ResolveCredentials() =>
			ResolveNegativeBalanceNodes(CredentialType.Amount)
			.ResolveNegativeBalanceNodes(CredentialType.Vsize)
			.ResolveZeroCredentials(CredentialType.Amount)
			.ResolveZeroCredentials(CredentialType.Vsize);
```

Reply:
normally I would agree, but I think generality is very unlikely to help us here, and even if it does become valuable, it's easily restored or this can be extended. i think in this readability/clarity is more important